packages/recipes/package.json

```diff
@@ -20,8 +20,10 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsup && pnpm build:recipes",
+    "build": "run-s build:package build:recipes build:format",
+    "build:package": "tsup",
     "build:recipes": "tsx ./scripts/build-recipes.ts",
+    "build:format": "prettier --config ../../.prettierrc --write .",
```

Comment:
I could have done the formatting manually in the build script but running prettier in isolation seems cleaner IMO